src/mapping/stake.ts

```diff
@@ -7,6 +7,20 @@ import {
 } from '@graphprotocol/graph-ts';
 import { getOrInitUser } from '../helper/initializer';
 import { updatePrice } from '../helper/price';
+import { StakeAmountChange } from '../../generated/schema';
+
+export function addStakeAmountChange(
```

Comment:
maybe move this to `src/helper/stake.ts` which fits better with current code structure.